ts/components/session/ActionsPanel.tsx

```diff
@@ -88,6 +99,14 @@ const Section = (props: { type: SectionType; avatarPath?: string }) => {
     );
   }
 
+  let iconColor = undefined;
+  if (type === SectionType.PathIndicator) {
+    // Set icon color based on result
+    iconColor = hasOnionPath ? Constants.UI.COLORS.GREEN : Constants.UI.COLORS.DANGER;
```

Comment:
rather use the theme here.
just like we have themify for css, in those components you can use

useTheme to get the current theme (and refresh on theme switch) and get the color to use with theme.colors.xxx